content/ADR-116-fast-bootstrapping.md

```diff
@@ -46,9 +46,13 @@ Instead of generating a complete snapshot every 6 hs, it will generate snapshots
   <img alt="Multiple snapshots" src="resources/RFC-16/multiple_snapshots.png"/>
 </figure>
 
+All the Catalyst will have the same initial timestamp *__t~initial~__* from which they will generate the same time ranges for the snapshots: _[t~initial~, t~1~], [t~1~, t~2~], ...,[t~n-1~, t~n~]_.
```

Comment:
```suggestion
All the Catalysts will have the same initial timestamp *__t~initial~__* from which they will generate the same time ranges for the snapshots: _[t~initial~, t~1~], [t~1~, t~2~], ...,[t~n-1~, t~n~]_.
```

Reply:
Have a look at how to use subscripts, because it's rendering incorrectly:
![Screenshot 2023-01-11 at 9 38 12 AM](https://user-images.githubusercontent.com/959563/211808207-e1857d1a-57c2-43cf-b26a-758e54044787.png)

Reply:
I'll review if subscripts can be done or just make them bold